Minesweeper.cpp

```
@@ -4,114 +4,310 @@
+					break;
 			}
 		}
-	}
 
-public:
-	Minesweeper(IntVector2 dimensions, GameBoard::Square sampleTile) : GameBoard(dimensions, sampleTile)
-	{
-		m_grid = GenerateBoard<TileGrid, Tile>(dimensions, sampleTile);
-	} 
+	public:
+		MinesweeperGrid(const IntVector2 dimensions, const Minesweeper::Tile sampleTile, const GameBoard::Grid<Tile>::AnchorPoints anchorPoint, const IntVector2 position)
```

cppcheck (2.10-2)

[performance] Function parameter 'sampleTile' should be passed by const reference.

(passedByValue)